go.mod

```diff
@@ -5,13 +5,14 @@ go 1.13
 require (
 	github.com/ElrondNetwork/arwen-wasm-vm v1.1.1
 	github.com/ElrondNetwork/concurrent-map v0.1.3
+	github.com/ElrondNetwork/elastic-indexer-go v0.0.0-20210212085149-2fa6f5938ae9
```

Comment:
why not a release?

Reply:
first I want some system tests with the new repository `master` branch and this branch  and after that, I will make a release

Reply:
ok, the release can be postponed whenever we are ready.